### DISCUSSION TITLE
Issue while using with typescript-eslint

### DISCUSSION THREAD

**Discussion by NaviTheCoderboi** [eslint/eslint#18509] [Q&A]
hi , guys why i getting this error even i don't have run it on eslint.config.js
```ts
Parsing error: ESLint was configured to run on `C:\Users\dell\Desktop\projects\node\windi-ui\packages\plugin\eslint.config.js` using `parserOptions.project`: C:\Users\dell\Desktop\projects\node\windi-ui\packages\plugin\tsconfig.json
```
tsconfig.json
```json
{
    "extends": "@repo/ts-config/base.tsconfig.json",
    "include": ["src/**/*"],
    "exclude": ["node_modules", "dist"]
}
```
eslint.config.js
```ts
import base from '@repo/eslint-config/base.js';

export default [
    ...base,
    {
        files: ['src/**/*']
    }
];
```

**Comment by Rec0iL99** [comment]
Hi @NaviTheCoderboi, the core team does not maintain the [`typescript-eslint`](https://github.com/typescript-eslint/typescript-eslint) project. Please open a discussion there for more help from the `typescript-eslint` maintainers.

**Comment by error-four-o-four** [comment]
Hey there,
typescript-eslint mentions this in their [FAQ](https://typescript-eslint.io/troubleshooting/#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file)

Basically i'd set the path of the 'tsconfig.json' as the value of property 'languageOptions.parserOptions.project'
